utils_cv/common/gpu.py

```diff
@@ -14,8 +14,30 @@ def which_processor():
         print("Cuda is not available. Fast.ai/Torch is using CPU")
 
 
+def is_linux():
+    """Returns if machine is running an Linux OS"""
+    return platform.system().lower() == "linux"
+
+
+def is_windows():
+    """Returns if machine is running an Windows OS"""
+    return platform.system().lower() == "windows"
+
+
 def linux_with_gpu():
     """Returns if machine is running an Linux OS and has a GPU"""
-    is_linux = platform.system().lower() == "linux"
     has_gpu = is_available()
-    return is_linux and has_gpu
+    return is_linux() and has_gpu
+
+
+def db_num_workers(non_windows_num_workers=16):
```

Comment:
[minor] following typing convention `non_windows_num_workers: int = 16`

Also, do we want to just set a static 16? people online seem to say for num_worker should = 4 * num_GPU .

Reply:
Thanks, changed to the recommended type convection. Leaving the number at 16 since that is the default fast.ai is using